server/app/mutations/volumes/plugin_install.rb

```diff
@@ -0,0 +1,41 @@
+module Volumes
+  class PluginInstall < Mutations::Command
+
+
+    required do
+      model :grid, class: Grid
+      string :name
+    end
+
+    optional do
+      string :alias_name
+      array :config do
+        string
+      end
+      string :label
+    end
+
+    def validate
+
+    end
+
+    def execute
+      # TODO
```

Comment:
Rcp push was more simple and straightforward. Yes, it has the drawback of installing plugins only on request and not automatically after a new node joins the grid.